### DISCUSSION TITLE
Quickstart

### DISCUSSION THREAD

**Discussion by Sherifx2** [NvChad/NvChad#2226] [Ideas]
I know this may seem silly, you just clone it into your nvim config. I counter with this- I've been doing this for over a year, and I am still an idiot. The greatest software I have ever used has always had really great documentation. The documentation was written as a vector of passion from the engineers to the end users. Its purpose was to show off this cool thing that they where building and explain how to use it for new people, but also be really efficient for more seasoned users to just jump in take what they need and go. I don't think it's a coincidence that those where my favorite projects. I would be scared off by anything less. It pains me to know all the cool stuff that I never even gave a chance because I thought it was scary programming stuff too big brained for me. I know the bar of entry may already seem very low, given that it's one command to get up and running but I also know that it would have scared me off if I hadent already learned how to use nvim now I'm kinda pissed off that it disables netrw by default, because I got used to it, it's more efficient for doing a lot of stuff, and there's literally no reason to disable it. (I can always enable it in my config this isn't a real criticism) but I wouldn't have even KNOWN about netrw if I started with this. Why not give that one command in the readme?

example:

Quickstart
install neovim per your package manager's discrepancy 
Create the ~/.config/nvim directory if not already there
run git clone https://github.com/NvChad/NvChad ~/.config/nvim 
run nvim . in the project you wish to work on and get started!

**Comment by antoineco** [comment]
(disclaimer: I'm not the project's maintainer)

Thanks for the opinion and suggestions 🙌  A few comments:

> install neovim

The prerequisites are listed here: https://nvchad.com/docs/quickstart/install

> Create the `~/.config/nvim` directory if not already there

The reason why it's not mentioned is because it's not necessary. Git does it for you.

> run `nvim .`

Not sure what you were going for here. NvChad is a _Neovim configuration_ as clearly stated both on GitHub and the website. It's obviously necessary to run Neovim in order to use Neovim.

> I'm kinda pissed off that it disables netrw by default

As any configuration written by someone who isn't you, it is likely to be opinionated in ways which aren't 100% aligned with your preferences. NvChad ships with NvimTree, which has similar functionalities and many more: integration with Git, integration with LSP, ... Using it instead of netrw felt like the right decision in a Neovim era.
